new-site/src/docs/elements/components/navigation/usage.mdx

```diff
@@ -0,0 +1,125 @@
+---
+slug: '/elements/components/navigation/code'
+title: 'Navigation - Usage'
+---
+
+import { Navigation } from 'hds-react';
+import { Link } from 'hds-react';
+
+## Usage
+
+### Example
+<Navigation
```

Comment:
These navigation examples would benefit from the new `PlaygroundPreview` component that Mika did, to add a border.